src/components/AddressItem/AddressesItem.js

```diff
@@ -0,0 +1,91 @@
+import { useState, useRef, useEffect } from 'react';
+import ButtonRound from 'components/UI/ButtonRound';
+import PropTypes from 'prop-types';
+import { toast } from 'react-toastify';
+import { useDeleteAddressMutation } from 'utils/rtk-query-addresses';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faTrashCan } from '@fortawesome/free-regular-svg-icons';
+import { useUpdateAddressMutation } from 'utils/rtk-query-addresses';
+
+const AddressesItem = ({ id, address }) => {
```

Comment:
I would think about renaming this component - it doean't describe what it really is

Reply:
Good point :+1: I changed the name for: *AddressLabel*. Would it be better?